src/styles/zIndex.js

```diff
@@ -0,0 +1,10 @@
+export default {
+  menu: 10,
+  appBar: 11,
+  leftNav: 12,
+  overlay: 13,
+  dialog: 14,
+  layer: 20,
+  popover: 20,
+  tooltip: 21,
```

Comment:
I think We must provide gaps for others to fit stuff between these.

This also broke LeftNav :D 

I have this proposal:
```js
menu: 100,
appBar: 110,
leftNavOverlay: 120,
leftNav: 130,
overlay: 140,
dialog: 150,
layer: 200,
popover: 210,
tooltip: 300,
```